### ISSUE TITLE
[BUG] ToU rates do not update

### ISSUE THREAD

**Issue by mcowger** [firstof9/ha-openei#29]
## Version of ha-openei
0.1.1

## Describe the bug
After installing and configuring against my plan, the initial load of utility rates is correct.  However, when the change in time happens to a higher window (say 2:59pm to 3:00pm), the sensor (in my case `sensor.pacific_gas_electric_co_current_energy_rate` does not update to the new rate.  

Expected behavior: value of `sensor.pacific_gas_electric_co_current_energy_rate` changes from 0.18 to 0.36
Actual behavior: value of `sensor.pacific_gas_electric_co_current_energy_rate` does not change after initial setup of the rate plan.

## Debug log

```text

2021-09-08 13:04:53 INFO (SyncWorker_1) [homeassistant.util.package] Attempting install of python-openei==0.1.9
2021-09-08 13:08:25 INFO (MainThread) [homeassistant.setup] Setting up openei
2021-09-08 13:08:25 INFO (MainThread) [homeassistant.setup] Setup of domain openei took 0.0 seconds
2021-09-08 13:08:25 INFO (MainThread) [custom_components.openei] 
-------------------------------------------------------------------
OpenEI
Version: 0.0.1
This is a custom integration!
If you have any issues with this you need to open an issue here:
https://github.com/firstof9/ha-openei/issues
-------------------------------------------------------------------

2021-09-08 13:08:25 INFO (MainThread) [homeassistant.components.binary_sensor] Setting up binary_sensor.openei
2021-09-08 13:08:25 INFO (MainThread) [homeassistant.components.sensor] Setting up sensor.openei
2021-09-08 13:08:25 INFO (MainThread) [homeassistant.helpers.entity_registry] Registered new binary_sensor.openei entity: binary_sensor.pacific_gas_electric_co_approval
2021-09-08 13:08:25 INFO (MainThread) [homeassistant.helpers.entity_registry] Registered new sensor.openei entity: sensor.pacific_gas_electric_co_current_energy_rate
2021-09-08 13:08:25 INFO (MainThread) [homeassistant.helpers.entity_registry] Registered new sensor.openei entity: sensor.pacific_gas_electric_co_distributed_generation
2021-09-08 13:08:25 INFO (MainThread) [homeassistant.helpers.entity_registry] Registered new sensor.openei entity: sensor.pacific_gas_electric_co_plan_name
2021-09-08 13:08:25 INFO (MainThread) [homeassistant.helpers.entity_registry] Registered new sensor.openei entity: sensor.pacific_gas_electric_co_all_listed_rates
```

If I restart HA when OpenEI starts it does successfully pick up the new current correct rate, but I wouldn't expect to have to do that.

This is the relevant rate plan: https://openei.org/apps/USURDB/rate/view/60a50f795457a33a3856fb5c#3__Energy

**Comment by firstof9** [comment]
Should be fixed in 0.1.2

**Comment by mcowger** [comment]
I can confirm that 0.1.2 corrects this behavior.  Thanks!
